core/src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -97,6 +98,7 @@ public ExportConfig(Map<String,Object> config) {
         this.config = config;
         this.streamStatements = toBoolean(config.get("streamStatements")) || toBoolean(config.get("stream"));
         this.writeNodeProperties = toBoolean(config.get("writeNodeProperties"));
+        this.ifNotExists = toBoolean(config.get("ifNotExists"));
```

Comment:
should the default be true? so it's more idempotent

Reply:
and then folks that have older versions can disable it.
or we introduce it here with default false and then change it in the next release (4.4)?